plataform_front_end/src/components/loader/loader.css

```diff
@@ -0,0 +1,37 @@
+@import url('https://fonts.googleapis.com/css?family= Baloo Da');
```

Comment:
```suggestion
```

Not necessary since you are already importing the font on [index.html](https://github.com/maclararose/Homer-Project/pull/1/files#diff-88de990e36d190d30ccfeaa34879de169785006eb26e7a8a57d5087378a53788R15)